packages/sanity/src/core/theme/index.ts

```diff
@@ -1,10 +1,7 @@
-import {buildTheme} from '@sanity/ui/theme'
-import {StudioTheme} from './types'
+import {RootTheme, buildTheme} from '@sanity/ui/theme'
 
 export * from './_legacy'
 export * from './types'
 
 /** @internal */
-export const defaultTheme: StudioTheme = buildTheme({
-  container: [320, 640, 960, 1280, 1600, 1920],
```

Comment:
removed this "customisation" as it won't be possible to change it through the studioThemeProvider, anyways this is the same value as the default Sanity UI value.